docs/src/main/sphinx/security/cli.rst

```diff
@@ -90,17 +90,21 @@ Additional Kerberos debugging information
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 You can enable additional Kerberos debugging information for the Trino CLI
-process by passing ``-Dsun.security.krb5.debug=true`` as a JVM argument, when
-starting the CLI process. Doing so requires invoking the CLI JAR via ``java``
-instead of running the self-executable JAR directly. The self-executable jar
-file cannot pass the option to the JVM.
+process by passing ``-Dsun.security.krb5.debug=true``,
+``-Dtrino.client.debugKerberos=true``, and
```

Comment:
How does `-Dtrino.client.debugKerberos=true` work?

Reply:
It turns on the debug option for the (Krb5LoginModule](https://docs.oracle.com/javase/8/docs/jre/api/security/jaas/spec/com/sun/security/auth/module/Krb5LoginModule.html): https://github.com/trinodb/trino/blob/b56ace136aaae60a5ae98984e97324924ad8379e/client/trino-client/src/main/java/io/trino/client/auth/kerberos/LoginBasedSubjectProvider.java#L96